third_party/move/move-model/src/builder/exp_builder.rs

```diff
@@ -2762,38 +2762,45 @@ impl<'env, 'translator, 'module_translator> ExpTranslator<'env, 'translator, 'mo
                     self.exit_scope();
                     self.new_bind_exp(loc, pat, binding, rest.into_exp())
                 },
-                Seq(exp) if items.len() > 1 => {
+                Seq(_) if items.len() > 1 => {
                     // This is an actual impl language sequence `s;rest`.
```

Comment:
PS. Perhaps pull this entire block into its own function. (Easy with RustRover.)

Reply:
done